settings.gradle

```diff
@@ -11,6 +11,7 @@ buildscript {
         jcenter()
     }
 }
+rootProject.name = "glean-core"
```

Comment:
Does this have any impact on consuming the library outside of the Glean repo? I remember messing with this property a while back and breaking things.

Reply:
That is indeed something I should check.

Reply:
(I have an abandoned commit around which just renames all glean-core mentions to glean, we could also just use that)

Reply:
(that would be cool!)

Reply:
Did you check this?